packages/components/package.json

```diff
@@ -1,89 +1,91 @@
-        "glob": "^7.1.5"
-    }
+  "name": "@deriv/components",
+  "version": "1.0.0",
+  "description": "deriv components",
+  "main": "src/index.js",
+  "private": true,
+  "engines": {
+    "node": "^14.17.1"
+  },
+  "scripts": {
+    "start": "rimraf lib && npm run test && npm run serve",
+    "serve": "rimraf lib && cross-env BUILD_MODE='serve' webpack --mode=production --progress --watch",
+    "build": "rimraf lib && npm run build:icons && webpack --mode=production --progress && npm run build:icons-manifest",
+    "build:icons": "node -e 'require(\"./utils/build-icons\").buildIcons()'",
+    "build:icons-manifest": "node -e 'require(\"./utils/build-icons-manifest\").buildIconsManifest()'",
+    "build:travis": "rimraf lib && webpack --mode=production",
+    "test": "node \"./src/components/icon/__tests__/icons.js\"",
+    "test:eslint": "eslint \"./src/**/*.?(js|jsx)\"",
+    "deploy": "echo \"No deploy specified\"",
+    "deploy:clean": "echo \"No deploy:clean specified\"",
+    "deploy:folder": "echo \"No deploy:folder specified\"",
+    "deploy:staging": "echo \"No deploy:staging specified\"",
+    "deploy:production": "echo \"No deploy:production specified\"",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
+  },
+  "author": "Deriv",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@babel/core": "^7.12.10",
+    "@babel/eslint-parser": "^7.17.0",
+    "@babel/preset-react": "^7.16.7",
+    "@storybook/addon-actions": "^6.5.10",
+    "@storybook/addon-essentials": "^6.5.10",
+    "@storybook/addon-interactions": "^6.5.10",
+    "@storybook/addon-links": "^6.5.10",
+    "@storybook/builder-webpack4": "^6.5.10",
+    "@storybook/builder-webpack5": "^6.5.10",
+    "@storybook/manager-webpack4": "^6.5.10",
+    "@storybook/manager-webpack5": "^6.5.10",
+    "@storybook/react": "^6.5.10",
+    "@storybook/testing-library": "^0.0.13",
+    "@testing-library/react": "^12.0.0",
+    "babel-loader": "^8.1.0",
+    "copy-webpack-plugin": "^9.0.1",
+    "copy-webpack-plugin-v6": "npm:copy-webpack-plugin@6",
+    "cross-env": "^5.2.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-config-binary": "^1.0.2",
+    "eslint-config-prettier": "^7.2.0",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-prettier": "^3.3.1",
+    "eslint-plugin-react": "^7.22.0",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "lint-staged": "^10.4.0",
+    "node-sass": "^7.0.1",
+    "react": "^16.14.0",
+    "sass-loader": "^12.6.0",
+    "sass-resources-loader": "^2.1.1",
+    "style-loader": "^1.2.1",
+    "svg-sprite-loader": "^5.2.1",
+    "svgo-loader": "^3.0.0",
+    "webpack": "^5.46.0",
+    "webpack-bundle-analyzer": "^4.3.0",
+    "webpack-cli": "^4.7.2"
+  },
+  "dependencies": {
+    "@contentpass/zxcvbn": "^4.4.3",
+    "@deriv/deriv-onboarding": "^1.0.0",
+    "@deriv/shared": "^1.0.0",
+    "@deriv/translations": "^1.0.0",
+    "@storybook/addon-info": "^5.3.21",
```

Comment:
are these two storybook packages needed? 
if they are need I think we should move them to `devDeps`.

Reply:
yes we need these 2 packages